docs/docs/community/security.md

```diff
@@ -0,0 +1,45 @@
+---
+id: security
+title: Security
+---
+
+:::note
+OAuth2 Proxy is a community project.
+Maintainers do not work on this project full time, and as such,
+while we endeavour to respond to disclosures as quickly as possible,
+this may take longer than in projects with corporate sponsorship.
+:::
+
+## Security Disclosures
+
+:::important
+If you believe you have found a vulnerability within OAuth2 Proxy or any of its
+dependencies, please do NOT open an issue on GitHub, please do NOT post any
```

Comment:
Worth mentioning not to open a PR with a proposed fix either?

Reply:
Yeah good point, I'll update